kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -370,6 +371,76 @@ def _create_kubernetes_state_prometheus_instance(self, instance):
                 }
             )
 
+        if join_standard_tags:
+            ksm_instance['label_joins'].update(
+                {
+                    "kube_pod_labels": {
+                        "labels_to_match": ["pod", "namespace"],
+                        "labels_to_get": [
+                            "label_tags_datadoghq_com_env",
+                            "label_tags_datadoghq_com_service",
+                            "label_tags_datadoghq_com_version"
+                        ]
+                    },
+                    "kube_deployment_labels": {
+                        "labels_to_match": ["deployment", "namespace"],
+                        "labels_to_get": [
+                            "label_tags_datadoghq_com_env",
+                            "label_tags_datadoghq_com_service",
+                            "label_tags_datadoghq_com_version"
+                        ]
+                    },
+                    "kube_replicaset_labels": {
+                        "labels_to_match": ["replicaset", "namespace"],
+                        "labels_to_get": [
+                            "label_tags_datadoghq_com_env",
+                            "label_tags_datadoghq_com_service",
+                            "label_tags_datadoghq_com_version"
+                        ]
+                    },
+                    "kube_daemonset_labels": {
+                        "labels_to_match": ["daemonset", "namespace"],
+                        "labels_to_get": [
+                            "label_tags_datadoghq_com_env",
+                            "label_tags_datadoghq_com_service",
+                            "label_tags_datadoghq_com_version"
+                        ]
+                    },
+                    "kube_statefulset_labels": {
+                        "labels_to_match": ["statefulset", "namespace"],
+                        "labels_to_get": [
+                            "label_tags_datadoghq_com_env",
+                            "label_tags_datadoghq_com_service",
+                            "label_tags_datadoghq_com_version"
+                        ]
+                    },
+                    "kube_job_labels": {
+                        "labels_to_match": ["job", "namespace"],
+                        "labels_to_get": [
+                            "label_tags_datadoghq_com_env",
+                            "label_tags_datadoghq_com_service",
+                            "label_tags_datadoghq_com_version"
+                        ]
+                    },
+                    "kube_cronjob_labels": {
+                        "labels_to_match": ["cronjob", "namespace"],
+                        "labels_to_get": [
+                            "label_tags_datadoghq_com_env",
+                            "label_tags_datadoghq_com_service",
+                            "label_tags_datadoghq_com_version"
+                        ]
+                    },
+                }
+            )
```

Comment:
should we declare these as global variables and refactor them a bit?  we're duplicating `labels_to_get`

Reply:
Great point, will refactor, thank you!